### ISSUE TITLE
Check whether or not GLA Toxin Tractor should have decreased min attack range

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#143]
Follow up to change: https://github.com/xezon/GeneralsGamePatch/pull/135
Maybe Min Attack Range should be tweaked a bit. Optional.

**Comment by xezon** [comment]
Checked. Min range 10 looks like so.

![shot_20230505_172425_3](https://user-images.githubusercontent.com/4720891/236501168-ec5f7bde-5e58-4509-a3ec-3900ec4884d4.jpg)

![shot_20230505_172835_5](https://user-images.githubusercontent.com/4720891/236501659-90fbc445-8fda-4117-bc73-715a1f780549.jpg)

It is odd that we standardized toxin tractor gun one way and dragon tank gun the other way, even though both units had the exact same issue.

Essentially Toxin Tractor was nerfed and Dragon Tank was buffed.

See

* #135
* #1588

**Comment by xezon** [comment]
Not clear to me if this needs touching again. Closing.

**Comment by commy2** [comment]
I would've removed Tractor min range from the upgraded versions. Clearly removal of min range was intentional, as the 0 is followed by a commented number. In 1.04, it makes no sense that Tox Toxin Tractor is worse at clearing buildings than regular Toxin Tractors.

**Comment by xezon** [comment]
Low firing range looks odd with that stream.

https://user-images.githubusercontent.com/4720891/236521530-cd4f83e2-c391-4459-9107-546af1ee2f6d.mp4

**Comment by commy2** [comment]
Yes, but that is how it looks in 1.04, and I think we shouldn't fix visuals if it negatively affects unit performance. Instead, the inconsistency / negative side effect of Anthrax Beta should be fixed by using the same min attack range (0) as the regular weapon.

Exactly the same as was done to the Dragon Tank.

**Comment by xezon** [comment]
We can do it if everyone agrees. It is noteworthy though that Toxin Tractor is better than Dragon Tank in that it has Attack Range 100 and Dragon has Attack Range 75.

**Comment by ExiLe-ZH** [comment]
I agree Tox's tractor shouldn't be worse than a Vanilla tractor at clearing buildings.

It's also ok compared to the dragon tank, the dragon has alot of other pros, like more HP and the ability to level structures very fast.
